dashboards-notebooks/.cypress/integration/ui.spec.js

```diff
@@ -130,6 +132,58 @@ describe('Testing notebooks table', () => {
   });
 });
 
+describe('Test reporting integration if plugin installed', () => {
+  beforeEach(() => {
+    cy.visit(`${Cypress.env('opensearchDashboards')}/app/notebooks-dashboards#`);
+    cy.get('.euiTableCellContent').contains(TEST_NOTEBOOK).click();
+    cy.wait(delay * 3);
+    cy.get('body').then($body => {
+      skipOn($body.find('#reportingActionsButton').length <= 0);
+    });
+  });
+
+  it('Create in-context PDF report from notebook', () => {
+    cy.get('#reportingActionsButton').click();
+    cy.wait(delay);
+    cy.get('button.euiContextMenuItem:nth-child(1)').contains('Download PDF').click();
+    cy.get('#downloadInProgressLoadingModal').should('exist');
+    cy.get('div.euiFlexGroup:nth-child(6) > div:nth-child(1) > button:nth-child(1)').click();
```

Comment:
is there a better way to get this element? if not could you add a comment on what it is

Reply:
Sure, I can actually remove it. I assume you are referring to line 150, that is just to click the `Cancel` button in the Loading Modal.